SDK/MobileCenter/Microsoft.Azure.Mobile.WinForms/Utils/ApplicationLifecycleHelper.cs

```diff
@@ -104,8 +111,13 @@ public bool Enabled
             }
         }
 
+        public bool HasShownWindow => started && !suspended;
```

Comment:
Should just map to started because the property only indicates whether UI has been shown at some point